app/assets/javascripts/components/student/pages/lessons_page.es6.jsx

```diff
@@ -1,12 +1,35 @@
 class LessonsPage extends React.Component {
 
+  static get propTypes() {
+    return {
+      studentId: React.PropTypes.number.isRequired,
+    }
+  }
+
   constructor() {
     super();
     this.state = {
       filter: "upcoming",
+      lessons: null,
     };
   }
 
+  componentDidMount() {
+    this.fetchLessons();
+  }
+
+  fetchLessons() {
+    const {studentId} = this.props
```

Comment:
this line seems pretty unnecessary, I think you can just call this.props.studentId directly in line 23